pkg/project/cloud/Taskfile.yml

```diff
@@ -149,6 +150,25 @@ tasks:
     cmds:
       - sh aemw package deploy --file "all/target/*.all-*.zip"
 
+  aem:check:
+    deps: [aem:author:check, aem:publish:check]
+
+  aem:author:check:
+    vars:
+      BASIC_AUTH:
+        sh: sh aemw config value -t "[[.Instance.Config.local_author.User]]:[[.Instance.Config.local_author.Password]]" --output-value value 2>&1
```

Comment:
bridging aem compose config with task tool

Reply:
TBH I have no clue why `2>&1` is required here but without it does not work; any ideas?

Reply:
A remark which is not related to this task itself - it's not clear for my what is the "--output-value value". Description of the command is not very helpful here

Reply:
hmm if empty then could be implicit `value` key used to make it shorter; this thing in intended to output only single value from command.

it could output only package details, instance details, bundle symbolic etc... this is kind of selector...